pandas/core/strings.py

```diff
@@ -1934,8 +1934,8 @@ def wrapper(self, *args, **kwargs):
             if self._inferred_dtype not in allowed_types:
                 msg = (
                     "Cannot use .str.{name} with values of inferred dtype "
-                    "{inf_type!r}.".format(
-                        name=func_name, inf_type=self._inferred_dtype
+                    f"{repr(self._inferred_dtype)}.".format(
+                        name=func_name
```

Comment:
i think the func_name needs to go into the previous line and the .format removed

Reply:
yes, i think you are right. I am changing it as you said.
btw, can you help me why there are failed check CI showing on linting ?

Reply:
I think you need to remove the closinig parenthesis on 1939